public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsRG.json

```diff
@@ -0,0 +1,900 @@
+            "resource": "/subscriptions/$sub",
+            "resultFormat": "table",
+            "workspace": ""
+          },
+          "azureMonitor": {
+            "aggOptions": [],
+            "dimensionFilter": "*",
+            "dimensionFilters": [],
+            "timeGrain": "auto",
+            "timeGrains": [],
+            "top": "10"
+          },
+          "azureResourceGraph": {
+            "resultFormat": "table"
+          },
+          "insightsAnalytics": {
+            "query": "",
+            "resultFormat": "time_series"
+          },
+          "queryType": "Azure Log Analytics",
+          "refId": "A",
+          "subscription": "4dc2cd39-7a89-43d8-bebe-8bb501359891",
```

Comment:
This seems not right to have subscription ids in a curated dashboard

Reply:
Not sure why they weren't stripped on export, I removed it